data/rooms/planet.txt

```diff
@@ -545,7 +545,7 @@ $s:autop1,skip1,mus17,scene10,wait15,mlog,tm210,tm211,tm212,mend,wait1,sctxt243,
   \"Computer, where are the survivors now?\""
 !T115"
   I don't know.  Deeper down, I think.  I am still
-  trying to access to the remote links down there,
+  trying to obtain access to the remote links down there,
```

Comment:
Thanks for catching this!

The text has to be wrapped to a specific width to fit in the dialogue box, and this new text would be too wide.  I suspect that I intended to write "trying to access the remote links down there," which should work fine.  Could you change it to that?